python/pyspark/sql/functions.py

```diff
@@ -267,8 +296,7 @@ def coalesce(*cols):
 
 @since(1.6)
 def corr(col1, col2):
-    """Returns a new :class:`Column` for the Pearson Correlation Coefficient for ``col1``
-    and ``col2``.
+    """Returns a new :class:`Column` for the Pearson Correlation Coefficient for `col1` and `col2`.
```

Comment:
Honestly, this is rather unnecessary change. Both before and after are fine and do not affect the documentation rendering.